src/panels/lovelace/hui-root.ts

```diff
@@ -614,7 +617,13 @@ class HUIRoot extends LitElement {
     } else {
       if (viewConfig.panel && viewConfig.cards && viewConfig.cards.length > 0) {
         view = createCardElement(viewConfig.cards[0]);
-        view.isPanel = true;
+        view = document.createElement("div");
+        view.id = "panel";
+        view.classList.add("panel");
```

Comment:
There is already a div with class=panel in the render method ?

Reply:
Yes, but since cast doesn’t have this div I can’t use it for the background, we could set the background on both that div and `hui-view`, but that will give problems with transparant backgrounds.

Reply:
Can we add it to cast? Is this better fixed by breaking up `hui-root` into a UI with tabs and a "content renderer" ? 

We can't have two divs with the same class, that is going to cause bugs eventually.

Reply:
So I used id's to apply the class, mean styles are duplicated a bit... Problem is that the theme is set in `hui-view` so we don't have the theme settings yet on the `#view` element. For a panel there was no theme set at all btw, that is now done in `hui-root`, but without watching for changes etc like in `hui-view` the best solution would be to have a panel element that deals with the theme properly and the styling.

Breaking up `hui-root` is a good idea, it is too big and does too much.